employee-rostering-gwtui/pom.xml

```diff
@@ -72,6 +118,40 @@
       <groupId>org.jboss.errai</groupId>
       <artifactId>errai-ioc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.errai</groupId>
+      <artifactId>errai-bus</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.errai</groupId>
+      <artifactId>errai-common</artifactId>
+    </dependency>    
+    <!-- kie-parent excludes this from errai-ioc, but it is required for GWT compile  -->
```

Comment:
Does this comment apply only to `com.google.inject:guice` or to other dependencies below as well? There were many new dependencies added - these are not transitive?

Reply:
it is related to the guice

Reply:
@mareknovotny Ok, so why so many new dependencies? Are they being used directly in this project and were not declared as such?